packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -104,12 +104,17 @@ const tokenTemplates: {
     FontSize: stringValueTemplate,
     TextTransform: stringValueTemplate,
     FontFamily: stringValueTemplate,
+    BoxShadow: stringValueTemplate,
+    OffsetX: stringValueTemplate,
+    OffsetY: stringValueTemplate,
     Font: fontTemplate,
     Size: stringValueTemplate,
     Width: stringValueTemplate,
     Height: stringValueTemplate,
     Delay: stringValueTemplate,
-    Padding: stringValueTemplate
+    Padding: stringValueTemplate,
+    Blur: stringValueTemplate,
```

Comment:
Same as above, Blur is just a css length value, so I'd go with the Size suffix. i.e. `elevation2BoxShadowBlurSize`